src/plugin/nodeSpec.ts

```diff
@@ -5,6 +5,9 @@ import type { FieldNameToValueMap } from "./fieldViews/helpers";
 import { fieldTypeToViewMap } from "./fieldViews/helpers";
 import type { FieldDescription, FieldDescriptions } from "./types/Element";
 
+export const elementTypeAttr = "pm-elements-element-type";
+export const fieldNameAttr = "pm-elements-field-name";
```

Comment:
I wonder if it might be nice to call these `pme-field-name` and `pme-element-type`?

Reply:
haha, I was having exactly the same thought!